src/components/image/image.tsx

```diff
@@ -126,18 +113,13 @@ export class Image
    */
   @Prop({ mutable: true }) width: string;
 
-  /**
-   * Emitted when the element is clicked.
-   */
-  @Event() gxClick: EventEmitter;
-
-  private handleClick(event: UIEvent) {
+  @Listen("click", { capture: true })
+  handleClick(event: UIEvent) {
+    event.preventDefault();
```

Comment:
Why are we preventing default?

Reply:
I include the preventDefault to prevent unexpected behavior... Do you suggest to remove that line?

Reply:
I'd remove it if it's just in case.

Reply:
removed! Check 09ae8dc